Source/NSPasteboard.m

```diff
@@ -553,6 +553,15 @@ to a filter implemented using the standard method (of
 #import "GNUstepGUI/GSServicesManager.h"
 #import "GNUstepGUI/GSPasteboardServer.h"
 
+/*
+ * FIXME
+ * We should learn to handle 
+ * NSPasteboardTypePNG
```

Comment:
I'm not sure what this refers to -- in what way do we need to learn to handle them? Do we need to learn how to paste an `NSPasteboardTypePNG` in `NSImageView` etc?

If so, is this the right place for this FIXME?